src/LiveComponent/assets/src/Rendering/ExternalMutationTracker.ts

```diff
@@ -189,16 +189,8 @@ export default class {
     private handleClassAttributeMutation(mutation: MutationRecord, elementChanges: ElementChanges) {
         const element = mutation.target as Element;
 
-        const previousValue = mutation.oldValue;
-        const previousValues = previousValue ? previousValue.split(' ') : [];
-        previousValues.forEach((value, index) => {
-            const trimmedValue = value.trim();
-            if (trimmedValue !== '') {
-                previousValues[index] = trimmedValue;
-            } else {
-                previousValues.splice(index, 1);
-            }
-        });
+        const previousValue = mutation.oldValue || '';
+        const previousValues = previousValue.match(/(\S+)/gu) || [];
```

Comment:
Could you keep juste this file 
(and the generated files in dists), 

and add just a test or two in `assets/test/Rendering/ExternalMutationTracker.test.js` ? 


I'm not sure we sure create and mainain an entire new "env" ... and i'm sure i don't want to need webpack locally :)) 

(i'm trying to slim the PR down to merge it rapidely :) )

Reply:
I added tests and made sure they break before the fixes are applied.